src/tasks/MonoTargetsTasks/ILStrip/ILStrip.cs

```diff
@@ -81,4 +131,99 @@ private bool StripAssembly(ITaskItem assemblyItem)
         return true;
     }
 
+    private bool TrimMethods(ITaskItem methodTokenFileItem, string assemblyPath)
+    {
+        string methodTokenFile = methodTokenFileItem.ItemSpec;
+        if (!File.Exists(methodTokenFile))
+        {
+            Log.LogMessage(MessageImportance.Low, $"[ILStrip] {methodTokenFile} doesn't exit.");
+            return true;
+        }
+        string[] log = File.ReadAllLines(methodTokenFile);
```

Comment:
How large is one of these files? Should we read it line by line instead of loading it into a large `string[]`?

Reply:
Fixed.